src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -295,6 +296,13 @@ private function loadUserContext(ContainerBuilder $container, XmlFileLoader $loa
             $container->getDefinition($this->getAlias().'.user_context.logout_handler')
                 ->replaceArgument(1, $config['user_identifier_headers'])
                 ->replaceArgument(2, $config['match']['accept']);
+
+            $container->register($this->getAlias().'.user_context.logout.handler.session', ContextInvalidationLogoutHandler::class)
```

Comment:
i'd prefer to declare this in the user context xml service definition file. https://symfony.com/doc/current/service_container/service_decoration.html also has an example for that.